src/models.ts

```diff
@@ -0,0 +1,26 @@
+import { BaseElement } from '@map-colonies/node-osm-elements';
+import { Feature, LineString, Point, Polygon } from 'geojson';
+
+export type Tags = Record<string, string> | undefined;
+export type OsmPoint = Feature<Point, Tags>;
+export type OsmLine = Feature<LineString, Tags>;
+export type OsmPolygon = Feature<Polygon, Tags>;
```

Comment:
types names are very confusing with the types from `node-osm-elements`

```suggestion
export type FlattenedGeoJSONPoint = Feature<Point, Tags>;
export type FlattenedGeoJSONLine = Feature<LineString, Tags>;
export type FlattenedGeoJSONPolygon = Feature<Polygon, Tags>;
```

Reply:
+1

Reply:
changed